pkg/query-service/app/metrics/v4/delta/timeseries.go

```diff
@@ -10,7 +10,7 @@ import (
 )
 
 // prepareTimeAggregationSubQueryTimeSeries builds the sub-query to be used for temporal aggregation
-func prepareTimeAggregationSubQueryTimeSeries(start, end, step int64, mq *v3.BuilderQuery) (string, error) {
+func prepareTimeAggregationSubQuery(start, end, step int64, mq *v3.BuilderQuery) (string, error) {
```

Comment:
this change should be at [cumulative/timeseries.go](https://github.com/SigNoz/signoz/tree/7d960b79dd34ae8413b2dab0a10224eec64b34a1/pkg/query-service/app/metrics/v4/cumulative) too?

Reply:
It will be done in upcoming https://github.com/SigNoz/signoz/pull/4169